data/actionsmap/yunohost.yml

```diff
@@ -660,6 +660,10 @@ app:
                 -f:
                     full: --file
                     help: Folder or tarball for upgrade
+                -F:
+                    full: --force
+                    help: Force the update, even though the app is up to date
+                    action: store_true
```

Comment:
I suggest a break change:

-f is used in install operation as in other part of the cli. I suggest to define -F for --file and -f for --force.

This change concerns packagers, and i prefer a break change for packager, than an inconsistency for final users

Reply:
N.B. : it's used here : https://github.com/YunoHost/package_check/blob/master/sub_scripts/testing_process.sh#L965 so gotta be careful

I'm okay with the change but maybe we can do this in a separate PR to move forward ...